client/cypress/integration/InfoBase/cypress-snapshots.js

```diff
@@ -127,20 +131,18 @@ module.exports = {
       "Tested on index-eng.html#infographic/gov/gov/covid": {
         "Axe violations allow list": {
           "aria-hidden-focus, serious": [
-            // Should be fixed by PR #1315
-            "<span class=\"link-unstyled\" tabindex=\"0\" aria-hidden=\"true\" data-toggle=\"tooltip\" data-ibtt-html=\"true\" data-ibtt-container=\"body\" data-ibtt-text=\"<p>",
-            "<span class=\"link-unstyled\" tabindex=\"0\" aria-hidden=\"true\" data-toggle=\"tooltip\" data-ibtt-html=\"true\" data-ibtt-container=\"body\" data-ibtt-text=\"<p>",
-            "<span class=\"link-unstyled\" tabindex=\"0\" aria-hidden=\"true\" data-toggle=\"tooltip\" data-ibtt-html=\"true\" data-ibtt-container=\"body\" data-ibtt-text=\"<p>"
+            "<span class=\"link-unstyled\" tabindex=\"0\" aria-hidden=\"true\" data-toggle=\"glossary_sidebar\" data-ibtt-html=\"true\" data-ibtt-container=\"body\" data-ibtt-text=\"<p>",
```

Comment:
Oops, yeah my bad. Thought I could just run the test for a specific route and it would only update that case without affecting the rest of the file

Reply:
All good. I left a line in the output that directs devs to update the file by running it locally. Could be clearer there, that's only for the worst case (something creates a false positive across a large number of routes) and requires care. Otherwise, updating lines manually works well.